content/en/users/data/management/datahub/_index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: EGI DataHub
+linkTitle: DataHub
+weight: 20
+type: docs
+description:
+  Discover, manage, and replicate data with EGI DataHub
+---
+
+## What is it?
+
+[EGI DataHub](https://datahub.egi.eu/) is a high-performance data management
+solution that offers **unified data access across globally distributed
+environments and multiple types of underlying storage**. It allows researchers
+to share, collaborate and perform computations on the stored data easily.
+
+Users can bring data close to their community or to the
+[compute facilities](../../../compute) they use, in order to exploit it efficiently.
+This is as simple as selecting which (subset of the) data should be available
+at which supporting provider.
+
+The main features of DataHub are:
+
+- Discovery of data spaces via a central portal
+- Policy based data access
+- Replication of data across providers for resiliency and availability purposes
+- Access to data from the [AppDB](../../../compute/cloud-compute/images/)
```

Comment:
at the the integration with AppDB has never been implemented, so given that we are revising this part i would remove it

Reply:
Done